testrail/api.py

```diff
@@ -175,17 +175,21 @@ def _refresh(self, ts):
 
     @classmethod
     def flush_cache(cls):
-        """ Set all cache objects to refresh
+        """ Set all cache objects to refresh the next time they are accessed
         """
+        def clear_ts(cache):
+            if 'ts' in cache:
```

Comment:
No that's fine. Not like keeping it is that big of a deal and then at least we know it keeps with existing functionality.